tests/Shared/GetTestProjects.proj

```diff
@@ -19,6 +19,8 @@
       <_TestProjectsToExclude Include="$(RepoRoot)tests\testproject\**\*Tests.csproj" />
       <_TestProjectsToExclude Include="$(RepoRoot)tests\TestingAppHost1\**\*Tests.csproj" />
 
+      <!-- This runs in a separate job -->
```

Comment:
We don't have to block this PR on it, but it would be good to have a README somewhere that shows how our matrix in GHA is composed, and which workflow controls which types of test projects.